### PR TITLE
[docs] Fix typo in migration guide

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: docs/data/migration/migration-pickers-v7/migration-pickers-v7.md [hunk 1]
```diff
@@ -315,18 +315,18 @@ The following variables were renamed to have a coherent `Picker` / `Pickers` pre
   + const translations = usePickerTranslations();
   ```
 
-  - `usePickersContext`
+  - `usePickerContext`
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v7/migration-pickers-v7.md:318]:
```suggestion
  - `usePickersContext`
```

File: docs/data/migration/migration-pickers-v7/migration-pickers-v7.md [hunk 2]
```diff
@@ -315,18 +315,18 @@ The following variables were renamed to have a coherent `Picker` / `Pickers` pre
...
 
   ```diff
-  - import { usePickersContext } from '@mui/x-date-pickers/hooks';
-  - import { usePickersContext } from '@mui/x-date-pickers';
-  - import { usePickersContext } from '@mui/x-date-pickers-pro';
+  - import { usePickerContext } from '@mui/x-date-pickers/hooks';
+  - import { usePickerContext } from '@mui/x-date-pickers';
+  - import { usePickerContext } from '@mui/x-date-pickers-pro';
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v7/migration-pickers-v7.md:323]:
```suggestion
  - import { usePickersContext } from '@mui/x-date-pickers/hooks';
  - import { usePickersContext } from '@mui/x-date-pickers';
  - import { usePickersContext } from '@mui/x-date-pickers-pro';
```

File: docs/data/migration/migration-pickers-v7/migration-pickers-v7.md [hunk 3]
```diff
@@ -315,18 +315,18 @@ The following variables were renamed to have a coherent `Picker` / `Pickers` pre
...
 
   + import { usePickerContext } from '@mui/x-date-pickers/hooks';
   + import { usePickerContext } from '@mui/x-date-pickers';
   + import { usePickerContext } from '@mui/x-date-pickers-pro';
 
-  - const pickersContext = usePickersContext();
+  - const pickersContext = usePickerContext();
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v7/migration-pickers-v7.md:329]:
```suggestion
  - const pickersContext = usePickersContext();
```

File: docs/data/migration/migration-pickers-v7/migration-pickers-v7.md (hunk 1 above)
Comment (flaviendelangle) [reply docs/data/migration/migration-pickers-v7/migration-pickers-v7.md:318]: :facepalm: sorry

**Review by LukasTy — APPROVED** [review]

**Pull request merged**